packages/osd-plugin-helpers/README.md

```diff
@@ -37,37 +38,44 @@ $ plugin-helpers help
   Usage: plugin-helpers [command] [options]
 
   Commands:
-      build
-        Copies files from the source into a zip archive that can be distributed for
-        installation into production OpenSearch Dashboards installs. The archive includes the non-
-        development npm dependencies and builds itself using raw files in the source
-        directory so make sure they are clean/up to date. The resulting archive can
-        be found at:
+    build
+      Copies files from the source into a zip archive that can be distributed for
+      installation into production OpenSearch Dashboards installs. The archive includes the non-
+      development npm dependencies and builds itself using raw files in the source
+      directory so make sure they are clean/up to date. The resulting archive can
+      be found at:
+
+        build/{plugin.id}-{opensearchDashboardsVersion}.zip
+
+      Options:
+        --skip-archive                       Don't create the zip file, just create the build/opensearch-dashboards directory
+        --opensearch-dashboards-version, -v  OpenSearch Dashboards version that the
+     
+    version
+      Updates the version to a provided parameter, or syncs it with the version of OpenSearch Dashboards
```

Comment:
It may be clear to users who are familiar with this tool, but I think we could be more explicit about which files are updated and where. Is it something I'd generally run before `build`, after running `build` (to update the version in just the artifacts), or is it more of a tool for development?

Reply:
assume this will change plugin package.json file which is also source code. a follow up CR is required to commit the change. If so, maybe provide a friendly remind to develop who used the tool.